### DISCUSSION TITLE
Want to add a multi-view display, as well as document links to help better proofreading work

### DISCUSSION THREAD

**Discussion by FinderBGG** [sumatrapdfreader/sumatrapdf#3562] [Ideas]
Want to add a multi-view display, as well as document links to help better proofreading work

**Comment by GitHubRulesOK** [comment]
SumatraPDF supports multiple views either
- multiple single panes
- multiple windows with multiple tabs

what it does not do is broken panes like a split view (used to be possible inside Firefox before 2015 Adobe SWF exploits insecurity overhauls) that is possible only in a few older browsers and problematic so was / is unsupported.
![untitled](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/3a78d033-4d6a-4ed1-b285-6b53b521ef04)

What is annoying is you can do that with Acrobat powered MS Edge but for "securities" sake it will not allow SumatraPDF to handle either view
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/7fc80364-967d-42d7-999f-570788d9644b)



Unclear what you mean by "links" as SumatraPDF will attempt to use any of the many types included that are valid, 
however as not a document editing application it cannot add to existing. (there was for a while experimental overlinking of all docs , actually worked quite well, but dropped as unwieldy)
